client/src/App.jsx

```diff
@@ -1,7 +1,145 @@
 import "./App.css";
+import { useState } from "react";
+import Paper from "@mui/material/Paper";
+import Typography from "@mui/material/Typography";
+import Container from "@mui/material/Container";
+import CreateContact from "./components/CreateContact";
+import ViewContact from "./components/ViewContact";
+import Contacts from "./components/Contacts";
 
 function App() {
-  return <></>;
+  const [selectedContactId, setSelectedContactId] = useState(null);
+  const [contacts, setContacts] = useState([]);
+
+  const containerStyles = {
+    display: "flex",
+    position: "relative",
+    alignItems: "stretch",
+    width: "100%",
+  };
+
+  const paperStyles = {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+  };
+
+  return (
+    <Container
+      sx={{
+        ...containerStyles,
+        flexDirection: "column",
+      }}
+    >
+      <Paper
+        elevation={1}
+        sx={{
+          ...paperStyles,
+          width: "1085px",
+          height: "50px",
+          marginBottom: "15px",
+          marginLeft: "45px",
+          "@media (max-width:800px": {
+            width: "500px",
+          },
+          "@media (max-width:400px)": {
+            width: "400px",
+          },
+        }}
+      >
+        <Typography
+          variant="h5"
+          sx={{
+            marginTop: "10px",
+            "@media (max-width:400px)": {
+              width: "400px",
+            },
+          }}
+        >
+          Contact List App
+        </Typography>
+      </Paper>
+      <Container
+        sx={{
+          ...containerStyles,
+          flexDirection: "row",
+          "@media (max-width:800px)": {
+            flexWrap: "wrap",
+            width: "100%",
+          },
+        }}
+      >
+        <Container
+          sx={{
+            ...containerStyles,
+            flexDirection: "column",
+            "@media (max-width:800px)": {
+              width: "100%",
+              marginBottom: "15px",
+            },
+          }}
+        >
```

Comment:
I'm seeing some nested `<Container>`s here. Note that Containers are basically Boxes with their content centered horizontally: https://mui.com/material-ui/react-container/

My understanding is they are used as the outermost container div element for your page and not really for children components. (So like you'll notice how many webpages have a lot of left and right-side padding for the entire page's content.) But please do prove me wrong if you see evidence somewhere suggesting otherwise! Apart from this usage preference issue, `Box` may be easier to use as it doesn't have styling applied to it already like `Container` does, potentially making overriding the default `Container` styles more troublesome.